front/app/containers/Admin/reporting/components/ReportBuilder/Widgets/IframeMultiloc/index.tsx

```diff
@@ -12,8 +12,6 @@ import { Multiloc } from 'typings';
 
 import useLocalize from 'hooks/useLocalize';
 
-import useReportDefaultPadding from 'containers/Admin/reporting/hooks/useReportDefaultPadding';
```

Comment:
Removed this hook because it was really complex and confusing, and in the end we don't really need it for the report builder